.github/workflows/release-versoview.yml

```diff
@@ -95,7 +95,14 @@ jobs:
           path: ./target/release/verso-x86_64-pc-windows-msvc.tar.gz
 
   build-mac:
-    runs-on: macos-latest
+    strategy: 
+      matrix:
+        include:
+          - platform: 'macos-14' 
+            args: '--target aarch64-apple-darwin'
```

Comment:
Shouldn't this be something like `aarch64-apple-darwin` and we plugin them into the `Build` job?